FireboltNETSDK/Client/FireboltConnectionStringBuilder.cs

```diff
@@ -33,20 +33,42 @@ public class FireboltConnectionStringBuilder : DbConnectionStringBuilder
         /// Gets or sets the name of the user.
         /// </summary>
         /// <returns>The name of the user.</returns>
-        public string ClientId
+        public string? UserName
         {
-            get => GetString(nameof(ClientId)) ?? throw new FireboltException("ClientId parameter is missing in the connection string");
+            get => GetString(nameof(UserName));// ?? (GetString(nameof(ClientId)) == null ? throw new FireboltException("Either ClientId or UserName must be provided") : null);
+            init => this[nameof(UserName)] = value;
+        }
```

Comment:
Here and in 4 other places in this file there's commented code. Did you mean to remove it?